CMakeLists.txt

```diff
@@ -92,6 +92,7 @@ include_directories(
   deps/GSL/include
   deps/outcome
   deps/sml/include
+  deps/sr25519-crust/include
```

Comment:
Please add imported target, similar to https://github.com/soramitsu/kagome/pull/117/files#diff-e23a3893b97f38154e05ff8b38f225eeR18